{
  try {
    template().requestBody("direct:getSObject" + suffix,"ILLEGAL_ID",Merchandise__c.class);
    fail("Expected SalesforceException");
  }
 catch (  CamelExecutionException e) {
    assertTrue(e.getCause() instanceof SalesforceException);
    assertTrue(e.getCause().getCause() instanceof SalesforceException);
    final SalesforceException cause=(SalesforceException)e.getCause().getCause();
    assertEquals(404,cause.getStatusCode());
    assertEquals(1,cause.getErrors().size());
    LOG.debug("Errors for 404: {}",cause.getErrors());
  }
}
